pages/api/book/[user].ts

```diff
@@ -63,10 +64,10 @@ const getLocationRequestFromIntegration = ({ location }: GetLocationRequestFromI
   return null;
 };
 
-export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+export default async function handler(req: NextApiRequest, res: NextApiResponse): Promise<void> {
```

Comment:
Looking at the docs (sorry this is unrelated to your PR) - I was just wondering why you needed to typehint a Promise<void> the handler function is not supposed to be async.

https://nextjs.org/docs/basic-features/typescript#api-routes